testutils/consul.go

```diff
@@ -28,8 +27,6 @@ type TestConsulServerConfig struct {
 
 // NewTestConsulServer starts a test Consul server as configured
 func NewTestConsulServer(tb testing.TB, config TestConsulServerConfig) *testutil.TestServer {
-	log.SetOutput(ioutil.Discard)
```

Comment:
The caller can set this instead of forcing all tests utilizing the helper to be muted.

Reply:
I changed it because it took me forever trying to debug something by running test and the verbose flag, and didn't realize it was not logging because I called test Consul server.

Just double checked the e2e CI job though, and definitely see a wall of logs 😂, so if this change is ok with everyone, I'll also look to remove `-v` from the job https://github.com/hashicorp/consul-terraform-sync/blob/master/Makefile#L53

Reply:
I found the common case was that I didn't want the logs and it was easier to keep them off and turn them back on when needed. The opposite could be true for CTS or other people just have different workflows that don't trigger that particular annoyance.

I do think some easy way to flip logging output on/off in testing would be the best long term solution. Maybe a testing CLI flag or a io.Writer var in testutils that is passed to SetOutput for tests... I'm not sure of the best answer but this does seem like something that would be very handy.

Reply:
I'm good with going with this change. It sounds like this was impacting seeing CTS and/or Consul logs for integration tests maybe? As far as e2e tests, I tested out this change locally with a failing test and all looked readable - I didn't see any Consul logs likely since they're still set to be discarded (~12 lines down) and CTS logs were handled as expected.